quickstarts/microsoft.signalrservice/signalr/main.bicep

```diff
@@ -0,0 +1,121 @@
+@description('The globally unique name of the SignalR resource to create.')
+param name string = uniqueString(resourceGroup().id)
+
+@description('Location for the SignalR resource.')
+param location string = resourceGroup().location
+
+@description('The pricing tier of the SignalR resource.')
+@allowed([
+  'Free_F1'
+  'Standard_S1'
```

Comment:
Thanks so much!  We released a new SKU `Premium_P1`, could you please include it?